resources/snippetpanel/snippetpanel.html

```diff
@@ -0,0 +1,315 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+        <link rel="stylesheet" href="vscode-resource:./snippetpanel.css">
+    </head>
+    <body>
+        <h2><span class="latex">L<sup>A</sup>T<sub>E</sub>X</span> Snippet Panel</h2>
+
+        <div class="tab">
+            <button class="tablinks active" onclick="openTab(event, 'Symbols')">Symbols</button>
+            <button class="tablinks" onclick="openTab(event, 'Tikz')">Tikz</button>
+        </div>
+
+        <div id="Symbols" class="tabcontent">
+            <div class="header">
+                <input type="text" name="search" id="symbolsearch" placeholder="Search" oninput="mathSymbolSearch()" />
+                <select onchange="mathSymbolSearch()"> </select>
+            </div>
+
+            <div class="grid"></div>
+        </div>
+
+        <div id="Tikz" class="tabcontent">
+            <h3>General</h3>
+            <div>
+                <a class="cmd" data-snippet="\n\coordinate (${1:id}) at ${2:position};$0" data-description="name a point X"
```

Comment:
Is there a reason you have to have hard-coded tikz snippets in the HTML file? How about defining them in a JSON file?

Reply:
That would be better. Not sure what a clean way to do the columns at the end would be though. If you have any suggestions for an elegant way to structure the JSON that would be great.

Reply:
Also with the little "require library" grouping I can't see a neat way.
I think the way it's done in HTML isn't too bad so unless anyone has a nice idea I'll just leave it this way for now.

```html
<a class="cmd"
data-snippet="\n\coordinate (${1:id}) at ${2:position};$0"
data-description="name a point X"
>\coordinate (X) at (3,5)</a>
```

![image](https://user-images.githubusercontent.com/20903656/61937962-9fbd2480-afc2-11e9-94ad-1b757cf8d301.png)